packages/pn-personagiuridica-webapp/src/components/Contacts/__test__/AddSpecialContactDialog.test.tsx

```diff
@@ -61,10 +61,7 @@ describe('test AddSpecialContactDialog', () => {
     expect(titleEl).toHaveTextContent(`special-contacts.modal-title`);
     const bodyEl = within(dialog).getByTestId('dialog-content');
     expect(bodyEl).toBeInTheDocument();
-    expect(bodyEl).toHaveTextContent(`special-contacts.pec`);
```

Comment:
Don't remove the check on input. Replace getById with queryById and check that the input is not shown

Reply:
doone